<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>DicomView</class>
    <widget class="QWidget" name="DicomView">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0">
            <property name="spacing">
                <number>0</number>
            </property>
            <property name="leftMargin">
                <number>0</number>
            </property>
            <property name="topMargin">
                <number>0</number>
            </property>
            <property name="rightMargin">
                <number>0</number>
            </property>
            <property name="bottomMargin">
                <number>0</number>
            </property>
            <item>
                <widget class="SokarUi::DicomToolBar" name="toolbar"/>
            </item>
            <item>
                <widget class="QSplitter" name="splitter">
                    <property name="orientation">
                        <enum>Qt::Horizontal</enum>
                    </property>
                    <widget class="Sokar::DicomGraphics" name="graphicsView"/>
                    <widget class="Sokar::FrameChooser" name="frameChooser">
                        <property name="minimumSize">
                            <size>
                                <width>45</width>
                                <height>0</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>260</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="widgetResizable" stdset="0">
                            <bool>true</bool>
                        </property>
                    </widget>
                </widget>
            </item>
            <item>
                <widget class="Sokar::MovieBar" name="movieBar" native="true"/>
            </item>
        </layout>
    </widget>
    <customwidgets>
        <customwidget>
            <class>Sokar::DicomGraphics</class>
            <extends>QGraphicsView</extends>
            <header>dicomview/graphics.h</header>
        </customwidget>
        <customwidget>
            <class>SokarUi::DicomToolBar</class>
            <extends>QToolBar</extends>
            <header>SokarUi/DicomToolBar.hpp</header>
        </customwidget>
        <customwidget>
            <class>Sokar::FrameChooser</class>
            <extends>QListWidget</extends>S
            <header>dicomview/framechooser.h</header>
        </customwidget>
        <customwidget>
            <class>Sokar::MovieBar</class>
            <extends>QWidget</extends>
            <header>dicomview/moviebar.h</header>
        </customwidget>
    </customwidgets>
    <resources/>
    <connections/>
</ui>
